src/app/core/parsers/zod-entity-parser.service.ts

```diff
@@ -0,0 +1,75 @@
+import { EntityParserService } from '../abstractions/entity-parser.service'
+import { AdLibPiece } from '../models/ad-lib-piece'
+import { BasicRundown } from '../models/basic-rundown'
+import { Part } from '../models/part'
+import { Piece } from '../models/piece'
+import { Rundown } from '../models/rundown'
+import { Segment } from '../models/segment'
+import * as zod from 'zod'
+
+
+export class ZodEntityParser implements EntityParserService {
+    private PIECE_PARSER = zod.object({
```

Comment:
These are instance variables. You are using are naming convention for constant values that are declared outside of the class. Please use our naming convention for instance variables or move them out to be constants.

Reply:
Sidenote: Instance variables should be declared as the first thing of the class. Not througout the class

Reply:
Both suggestions are applied. Both for `ZodEntityParser` and `ZodRundownEventParser`.